### ISSUE TITLE
Add a hyperlink on the carousel slide to make it clickable

### ISSUE THREAD

**Issue by edrthomas** [Dash-Industry-Forum/Dash-Industry-Forum.github.io#3]

**Comment by wilaw** [comment]
I made  a start at this.

1. To each item, in the  Dash-Industry-Forum.github.io/data/carousel/ directory, I added a new "link" tag. For example

`position: 0
title: "Guidelines for Implementation: DASH-IF Interoperability Points V4.3 released"
text: "
      - On-Demand and Mixed Services\n
      - HDR Dynamic Metadata and other Improvements\n
      - January 2019"
link: "//dashif.org/guidelines"`

2. To the  Dash-Industry-Forum.github.io/layouts/partials/carousel.html definition, I added a new `<a>` tag around the title.

3. In Dash-Industry-Forum.github.io/src/css/ I added `.carousel-slide-title:hover {text-decoration: none; color: yellow; font-weight: bold;}`

The current solution only makes the title clickable, but it is workable. As an improvement, it would be good to make all the elements of the carousel item (title, picture, text) clickable.
